.CI/create_feedstocks.py

```diff
@@ -120,7 +120,7 @@ def write_token(name, token):
         # Break the previous loop to allow the TravisCI registering to take place only once per function call.
         # Without this, intermittent failiures to synch the TravisCI repos ensue.
         for feedstock_dir, name, recipe_dir in feedstock_dirs:
-            subprocess.check_call(['conda', 'smithy', 'register-ci', feedstock_dir] + owner_info)
+            subprocess.check_call(['conda', 'smithy', 'register-ci', '--feedstock_directory', feedstock_dir] + owner_info)
```

Comment:
Maybe use `subprocess.check_output(...)` https://docs.python.org/2/library/subprocess.html#subprocess.check_output and then wrap the whole thing in a 

``` 
try:
   ... all the subprocess calls
except subprocess.CalledProcessError as e:
    print("Failed with returncode: %s\nOutput:\n%s\n" %(e.returncode, e.output))
    sys.exit(-1)
```

Sorry for all the breakage: when I did the changes, I didn't even know the `.CI` subdirectory existed... :-/